_includes/modal-speaker-1.html

```diff
@@ -0,0 +1,49 @@
+<div class="modal modal-speaker modal-speaker-1" id="modal-speaker-1" tabindex="-1" role="dialog"
```

Comment:
This file doesn't seem to be used. Can you remove it from the changes ?

Reply:
I do not use this code. I created this file as a copy of the source code in case you need it.
If it is not needed, it should be removed so that it does not confuse everyone.

Reply:
Done.